src/duplicacy_backupmanager.go

```diff
@@ -1689,7 +1696,7 @@ func (manager *BackupManager) CopySnapshots(otherManager *BackupManager, snapsho
 
     for _, snapshot := range snapshots {
         if revisionMap[snapshot.ID][snapshot.Revision] == false {
-            continue
+          continue
```

Comment:
What's this doing here? It looks like a formatting-only change that makes it not line up with the rest of the indentation levels.